repo/packages/C/concord/0/config.json

```diff
@@ -0,0 +1,51 @@
+{
+  "type": "object",
+  "properties": {
+    "mesos": {
+      "description": "Mesos configuration properties",
+      "type": "object",
+      "properties": {
+	"master": {
+	  "default": "zk://master.mesos:2181/mesos",
+	  "description": "The URL of the mesos master. The format is a comma-delimited list of hosts like this zk://host1:port,host2:port/mesos",
+	  "type": "string"
+	}
+      }
+    },
+    "concord": {
+      "description": "Concord configuration properties",
+      "type": "object",
+      "properties": {
+        "zookeeper": {
+	  "default": "zk://master.mesos:2181/concord",
```

Comment:
It would be nice to make it easier for people to install multiple versions of this package on dc/os if they want to. Providing a default value here lets people shoot themselves in the foot more easier than not.

two suggestions:
(1) define a `cluster-name` setting, with a default of `concord` or some such
(2) instead of providing a default value here for `zookeeper`, you could handle the default case in the marathon JSON template, at which point you could also use the `cluster-name` setting

see cassandra for an example of this: https://github.com/mesosphere/universe/blob/version-2.x/repo/packages/C/cassandra/1/marathon.json.mustache#L62

The idea is that when people want to install multiple versions of this package, they can just change the `cluster-name` setting without worrying about the `zookeeper` settings.